### ISSUE TITLE
omega

### ISSUE THREAD

**Issue by keplx** [linknetx/skin.estuary-plus#1]
is there an omega compliant version?

**Comment by linknetx** [comment]
> is there an omega compliant version?

No, I stopped work on this quite a while ago and I'm afraid there are no plans to update it.

**Comment by linknetx** [comment]
Change of plan, a new version for Omega can be found here:
https://forum.kodi.tv/showthread.php?tid=379877
and here:
https://github.com/linknetx/skin.estuary-extra
